sdk/src/androidTest/java/com/mapbox/maps/StyleLoadTest.kt

```diff
@@ -69,8 +69,7 @@ class StyleLoadTest {
       }
     }
     countDownLatch.await(10, TimeUnit.SECONDS)
-    // we should notify only once that style is loaded
-    Assert.assertEquals(1, styleLoadedCount)
+    Assert.assertEquals(2, styleLoadedCount)
```

Comment:
why did it change?

Reply:
I would prefer changing the test setup itself to accommodate the changes with lifecycle plugin. The test was written to validate style load. Validating that logic kind falls apart if we are loading 2 styles.